src/test/scala/org/camunda/feel/impl/interpreter/InterpreterFunctionTest.scala

```diff
@@ -179,4 +179,8 @@ class InterpreterFunctionTest
     eval("round(3.2)", functions = functions) should be(ValNumber(3))
   }
 
+  it should "be properly evaluated when parameters contain whitespaces" in {
+    eval("""number(from: "1.000.000,01", decimal separator:",", grouping separator:".")""") should be(ValNumber(1_000_000.01))
```

Comment:
:wrench: We could create another test case for a user-defined function.

Like [here](https://github.com/camunda/feel-scala/blob/master/src/test/scala/org/camunda/feel/impl/interpreter/InterpreterFunctionTest.scala#L48-L55) but with a function parameter that has whitespace. Take the function definition as an argument and invoke the function in the test case.

Reply:
Ok! Do you mean anything like:

```scala
    val functions =
      Map("f" -> eval("""function(test name) test name + 1""").asInstanceOf[ValFunction])
```

?

Reply:
Yes :+1: But I guess that you need to escape the name in the function body. Names with whitespaces are not fully supported :sweat_smile: 

```
    val functions =
      Map("f" -> eval("""function(test name) `test name` + 1""").asInstanceOf[ValFunction])

    eval("f(test name: 1)", functions = functions) should be(ValNumber(2))
```